lib/capybara/selenium/driver.rb

```diff
@@ -116,7 +116,7 @@ def reset!
 
   ##
   #
-  # Webdriver supports frame name, id, index(zero-based) or {Capybara::Element} to find iframe
+  # Webdriver supports frame name, id, index(zero-based) or {Capybara::Node::Base} to find iframe
```

Comment:
Lets leave this as Capybara::Node::Element,  while its technically true that it can take a Base, Base isn't really meant for public use and is just used as the superclass for Element and Document. Document makes no sense being passed in as a frame element.